src/main/webapp/js/user-page-loader.js

```diff
@@ -90,9 +91,25 @@ function buildMessageDiv(message) {
   return messageDiv;
 }
 
+// TODO:Understand what in the world this means
```

Comment:
- Remove TODO comment.
- nit: Add space between ')' and '{' to mirror the style in the rest of this code.

Arrows are syntactic sugar for a function declaration.
(response) => { return response.txt(); } is just a shorter way of saying

function (response) { return response.txt(); } 

It is typically used in situations like this where you have an unnamed callback function.

Reply:
Comment cleanup in aadbb898961b2f695c4e21581f1c978bbab78f93. Thank you for the information, I'll look further into it, as this concept strikes me as very strange